go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/edgexfoundry/go-mod-bootstrap/v3
 
 go 1.18
 
+replace github.com/edgexfoundry/go-mod-messaging/v3 => ../go-mod-messaging
```

Comment:
Shouldn't this use go-mod-messaging/v3 and not your local version?

Reply:
I just read the PR description. I assume this will be changed after [edgexfoundry/go-mod-messaging#210]( https://github.com/edgexfoundry/go-mod-messaging/pull/210) has been merged

Reply:
> I just read the PR description. I assume this will be changed after [edgexfoundry/go-mod-messaging#210](https://github.com/edgexfoundry/go-mod-messaging/pull/210) has been merged

Correct. This is why the PR is still in draft.